laravel/database/factories/SlotFactory.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use App\Models\Schedule;
+use App\Models\Slot;
+use Faker\Generator as Faker;
+
+$factory->define(Slot::class, function (Faker $faker, array $data)
+{
+    if(env('APP_DEBUG') && !isset($data['schedule_id']))
+    {
+        Log::warning("Using Factory[Slot] without setting schedule_id");
+    }
+
+    $row_min = 1;
```

Comment:
Rows are supposed to be created sequentially, based on the number of volunteers selected in the UI. This probably isn't a big deal when writing a basic test, but might be something to keep in mind for the future.

Also, you can automatically generate slots for a given schedule by calling the `Slot::generate($schedule)` function. https://github.com/playasoft/volunteers/blob/ccc6e5d98d4d7bd4b5358116d66c2859da0aa342/laravel/app/Models/Slot.php#L56